packages/lesswrong/components/ea-forum/EABestOfPage.tsx

```diff
@@ -253,12 +256,12 @@ const EABestOfPage = ({ classes }: { classes: ClassesType }) => {
               <div>
                 <h2 className={classes.heading}>Highlights this month</h2>
                 <div className={classes.listSection}>
-                  {monthlyHighlights?.map((post) => (
+                  {monthlyHighlights?.map((post, i) => (
                     <EAPostsItem
                       key={post._id}
                       post={post}
                       className={classes.postsItem}
-                      showIcons={false}
+                      curatedIconLeft={i < currentCuratedPostCount}
```

Comment:
Why can't you use `post.curatedDate` here?

Reply:
Because we only want to show the icon on the left for posts that are "currently curated" and as far as I can tell from the code, the only thing that defines "currently curated" as opposed to "ever currated" is that the last 3 curated posts (or 2 for logged out users) are "current".